### PR TITLE
Issue #30: add admin panel for requests to space api

### PR DESCRIPTION
**Pull request by thimovss**

### CODE REVIEW

**Comment by thimovss** [comment]
fireshot screen capture 033 - membership management system - localhost_3000_admi

**Comment by rngadam** [comment]
- Back history doesn't work for admin sub-menu
- use of collectionTable

File: app/imports/lib/collections/OrgLogs.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    var successLog = logs.createNewSuccessLog(TEST_URL, {payload: 'some string'});
+
+    logs.createOrUpdateLog(errorLog);
+    logs.createOrUpdateLog(successLog);
+
+    var entry = logs.OrgLogs.findOne({url: TEST_URL});
+    chai.assert.equal(entry.statusCode, 200);
+    chai.assert.instanceOf(entry.lastSuccess, Date);
+    chai.assert.equal(entry.error, null);
+    chai.assert.equal(entry.data.payload, 'some string');
```
Comment (thimovss) [inline app/imports/lib/collections/OrgLogs.test.js:51]: check for timestamp

File: app/lib/collections/OrgLogs.js [hunk 2]
```diff
@@ -1 +1,85 @@
...
+  if (e.response) {
+      statusCode = e.response.statusCode;
+  } 
+  return { 
+    timestamp: new Date(),
+    statusCode: statusCode, 
+    url: url, 
+    data: null, 
+    // clone to plain object to get rid of tricky native properties
+    error: JSON.parse(JSON.stringify(e))
```
Comment (thimovss) [inline app/lib/collections/OrgLogs.js:65]: > Beware using the JSON.parse(JSON.stringify(obj)) method on Date objects - JSON.stringify(new Date()) returns a string representation of the date in ISO format, which JSON.parse() doesn't convert back to a Date object.

[http://stackoverflow.com/questions/122102/what-is-the-most-efficient-way-to-deep-clone-an-object-in-javascript/5344074#5344074](source)

File: app/imports/lib/collections/OrgLogs.test.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+  skip_client('success count updated', function() {
+    var newLog = logs.createNewSuccessLog(TEST_URL, {payload: 'some string'});
+    logs.createOrUpdateLog(newLog);
+    logs.createOrUpdateLog(newLog);
+    logs.createOrUpdateLog(newLog);
+    var entry = logs.OrgLogs.findOne({url: TEST_URL});
+    chai.assert.equal(entry.successCount, 3);
+  });
+});
```
Comment (thimovss) [inline app/imports/lib/collections/OrgLogs.test.js:100]: Add test for "successful log followed by error log"

**Comment by thimovss** [comment]
- [x] Column widths are inconsistent https://github.com/coderbunker/candra/issues/48
- [x] You can sort on the Full Log button
- [x] There are CRUD operations that are not needed
- [x] Back history still doesn't work: https://github.com/coderbunker/candra/issues/49
- [x] add the failureCount,successCount and lastSuccess to the table (Maybe) 
- [x] format the timestamp/lastSuccess using angular-moment: https://github.com/coderbunker/candra/issues/50

File: app/imports/lib/collections/OrgLogs.test.js (hunk 1 above)
Comment (rngadam) [reply app/imports/lib/collections/OrgLogs.test.js:51]: done

File: app/lib/collections/OrgLogs.js (hunk 2 above)
Comment (rngadam) [reply app/lib/collections/OrgLogs.js:65]: it's not a date object... so...

File: app/imports/lib/collections/OrgLogs.test.js (hunk 3 above)
Comment (rngadam) [reply app/imports/lib/collections/OrgLogs.test.js:100]: done

**Pull request merged**